core/src/main/java/uk/ac/bbsrc/tgac/miso/core/data/impl/ContainerQC.java

```diff
@@ -0,0 +1,35 @@
+package uk.ac.bbsrc.tgac.miso.core.data.impl;
+
+import javax.persistence.Entity;
+import javax.persistence.JoinColumn;
+import javax.persistence.ManyToOne;
+import javax.persistence.Table;
+
+import uk.ac.bbsrc.tgac.miso.core.data.QC;
+import uk.ac.bbsrc.tgac.miso.core.data.QualityControllable;
+import uk.ac.bbsrc.tgac.miso.core.data.SequencerPartitionContainer;
+
+@Entity
+@Table(name = "ContainerQC")
+public class ContainerQC extends QC {
+
+  private static final long serialVersionUID = 1L;
+
+  @ManyToOne(targetEntity = SequencerPartitionContainerImpl.class)
+  @JoinColumn(name = "containerId")
+  private SequencerPartitionContainer container;
+
+  public SequencerPartitionContainer getContainer() {
+    return container;
+  }
+
+  public void setContainer(SequencerPartitionContainer container) {
+    this.container = container;
+  }
+
+  @Override
+  public QualityControllable<?> getEntity() {
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Update this method so that its implementation is not identical to "getContainer" on line 22. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS4144)